### ISSUE TITLE
Re-center to OP

### ISSUE THREAD

**Issue by lehitoskin** [jaxry/bubblechan#13]
I think there needs to be a key that brings you back to the OP (pressing Home, perhaps?). This would help in situations where there are several branching conversations and you need to scroll a lot to get back there (because zooming out is for nerds).

**Comment by jaxry** [comment]
Good idea. But I've noticed that it's often a post other than the OP which branches off into several conversations. In this case centering back on the OP wouldn't help much. What do you think about having it so that right clicking a node 'marks' it, and right clicking thereafter on empty space centers you back to the last marked node?

**Comment by lehitoskin** [comment]
That sounds perfect!
